modules/doc/content/getting_started/installation/manual_gcc.md

```diff
@@ -1,64 +1,40 @@
 ## GCC
 
-We need a modern C++17 capable compiler. Our minimum requirements are: GCC [!package!minimum_gcc], Clang [!package!minimum_clang]. This section will focus on building a GCC [!package!gcc] compiler stack.
+We need a modern C++17 capable compiler. Our minimum requirements are: GCC [!package!minimum_gcc],
+Clang [!package!minimum_clang]. This section will focus on building a GCC [!package!gcc] compiler
+stack.
 
-What version of GCC do we have?
-
-```bash
-gcc --version
-
-gcc (GCC) 4.8.5 20150623 (Red Hat 4.8.5-4)
-Copyright (C) 2015 Free Software Foundation, Inc.
-This is free software; see the source for copying conditions.  There is NO
-warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
-```
-
-If your version is less than [!package!minimum_gcc], you will need to build a newer version. If your version is at or
-greater than [!package!minimum_gcc], you have the option of skipping the GCC section.
+Obtain GCC source:
 
 !package! code
-cd $STACK_SRC
 curl -L -O http://mirrors.concertpass.com/gcc/releases/gcc-__GCC__/gcc-__GCC__.tar.gz
 tar -xf gcc-__GCC__.tar.gz -C .
 !package-end!
 
 Obtain GCC pre-reqs:
 
 !package! code max-height=450
-cd $STACK_SRC/gcc-__GCC__
+cd gcc-10.4.0
```

Comment:
do you want to remove the variable? seems you are adding more text variables in this PR